charts/ceph-csi-cephfs/values.yaml

```diff
@@ -300,6 +313,18 @@ storageClass:
   # If omitted, defaults to "csi-vol-".
   # volumeNamePrefix: "foo-bar-"
   volumeNamePrefix: ""
+
+  # (optional) Instruct the plugin it has to encrypt the volume
+  # By default it is disabled. Valid values are "true" or "false".
+  # A string is expected here, i.e. "true", not true.
+  # encrypted: "true"
+  encrypted: ""
+
+  # (optional) Use external key management system for encryption passphrases by
+  # specifying a unique ID matching KMS ConfigMap. The ID is only used for
+  # correlation to configmap entry.
+  encryptionKMSID: ""
+  
```

Comment:
there seems to be a trailing space here ([ci job](https://github.com/ceph/ceph-csi/actions/runs/8514964926/job/23336366170?pr=4531#step:3:1116))